### PR TITLE
json-write tests

### PR DESCRIPTION
**Pull request by edporras**
We broke json-writes with the switch to charred 😬

Pushed a fix and released a new version yesterday. This adds a test to verify the behavior.

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -29,5 +29,6 @@
   :profiles {:dev {:global-vars {*warn-on-reflection* true}
                    :source-paths ["dev"]
                    :resource-paths ["test-resources"]
-                   :dependencies [[org.clojure/tools.namespace "1.3.0"]]}}
+                   :dependencies [[org.clojure/data.json "2.4.0"]
```
Comment (mthl) [inline project.clj:32]: I would rather prefer not relying on a second JSON library to check the behavior of the one we are using. Would it be possible to check output instead by slurping the file (and optionally decompressing it) and comparing it with a string encoded JSON ?

File: test/tools/io_test.clj [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+       (catch IllegalArgumentException e
+         (println (str "Error parsing file " path "\n" (.getMessage e)))
+         :json-parse-error)))))
+
+(deftest write-jsons-file-test
+  (let [data [{:a 1 :b 2.3 :c "c1"}
+              {:a 2 :b 12.3 :c "c2"}]]
+    (testing "nominal case"
+      (sut/with-tempdir [tmp-dir]
+        (let [file-path (sut/join-path tmp-dir "test.json")]
```
Comment (mthl) [inline test/tools/io_test.clj:31]: should be "test.jsons"

File: test/tools/io_test.clj [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+          (sut/write-jsons-file file-path data)
+
+          (let [rslt (read-json-file file-path)]
+            (is (= data rslt))))))
+
+    (testing "with compression"
+      (sut/with-tempdir [tmp-dir]
+        (let [file-path (sut/join-path tmp-dir "test.json.gz")]
```
Comment (mthl) [inline test/tools/io_test.clj:39]: should be "test.jsons.gz"

File: project.clj (hunk 1 above)
Comment (edporras) [reply project.clj]: done

**Pull request merged**